src/scripts/breeding/Egg.ts

```diff
@@ -47,9 +63,7 @@ class Egg implements Saveable {
         this.pokemon = json['pokemon'];
         this.type = json['type'];
         this.notified = json['boolean'];
```

Comment:
I know this isn't a part of your PR, but shouldn't this be:
```suggestion
        this.notified = json['notified'];
```